content/design-systems-for-developers/react/pt/build.md

```diff
@@ -0,0 +1,238 @@
+
+```javascript
+import '@storybook/addon-actions/register';
+import '@storybook/addon-links/register';
+import '@storybook/addon-storysource/register';
+```
+
+E atualize a configuração do webpack que se encontra em `.storybook/webpack.config.js`: 
+
+```javascript
+module.exports = function({ config }) {
+ config.module.rules.unshift({
+   test: /\.stories\.jsx?$/,
+   loaders: [require.resolve('@storybook/addon-storysource/loader')],
+   enforce: 'pre',
+ });
+
+ return config;
+};
+```
+No Storybook este fluxo de trabalho assemelha-se ao seguinte:
+
+![O extra Storysource](/design-systems-for-developers/storybook-addon-storysource.png)
+
+<h4>Knobs fazer testes de stress aos componentes</h4>
+
+O [extra Knobs](https://github.com/storybookjs/storybook/tree/next/addons/knobs) permite que interaja dinamicamente com os adereços (props na forma original) do componente no IU do Storybook. O Knobs permite que sejam fornecidos múltiplos valores ao adereço (prop na forma original) de um componente e ajustá-las de acordo via IU. O que ajuda aos criadores de sistemas de design a fazerem testes de stress aos inputs dos componentes ajustando enfim, knobs. Mas também fornece aos consumidores de sistemas de design uma forma de testar os componentes antes de serem integrados, de forma a que possam compreender como cada adereço (prop na forma original) afeta o componente.
+
+Vamos ver como isto funciona, através da configuraçao dos knobs no componente `Avatar`:
+
+```bash
+yarn add --dev @storybook/addon-knobs
+```
+
+Registe-o em `.storybook/addons.js`:
+
+```javascript
+import '@storybook/addon-actions/register';
+import '@storybook/addon-links/register';
+import '@storybook/addon-storysource/register';
+import '@storybook/addon-knobs/register';
+```
+
+Adicione uma estória que usa os knobs em  `src/Avatar.stories.js`:
+
+```javascript
+import React from 'react';
+import { withKnobs, select, boolean } from '@storybook/addon-knobs';
+
+// …
+
+export const knobs = () => (
+  <Avatar
+    loading={boolean('Loading')}
+    size={select('Size', ['tiny', 'small', 'medium', 'large'])}
+    username="Dominic Nguyen"
+    src="https://avatars2.githubusercontent.com/u/263385"
+  />
+);
+
+knobs.story = {
+  decorators: [withKnobs],
+};
+```
+
+No painel de extras, repare na tab Knobs. Instrumentalizámos o elemento de seleção "Size" de forma a permitir alternar entre os vários tamanhos suportados para o Avatar, `tiny`, `small`, `medium` e finalmente `large`. Pode instrumentalizar quaisquer outros adereços (props na forma original) com os knobs, também criar um "recreio" interativo para o componente.
+
+![Extra knobs Storybook](/design-systems-for-developers/storybook-addon-knobs.gif)
+
+Posto isto, os knobs não substituem as estórias. São fantásticos para explorar casos extremos associados aos componentes. Mas para ilustrar os casos pretendidos, são usadas estórias.
+
+Tanto o extra Docs, como o o extra Accessbility serão apresentados em capítulos posteriores.
```

Comment:
I would suggest to update this to `Tanto o extra Docs, como o extra Accessibility serão apresentados em capítulos posteriores.`

Reply:
The wording was corrected in the latest commit. Thank you for spotting this one.